client/layout/universal-navbar-footer/index.tsx

```diff
@@ -63,53 +70,73 @@ const UniversalNavbarFooter = () => {
 							<h3>{ translate( 'Features' ) }</h3>
 							<ul>
 								<li>
-									<a href="https://wordpress.com/features/" title="WordPress.com Features">
+									<a
+										href="https://wordpress.com/features/"
+										title="WordPress.com Features"
```

Comment:
I know this is pre-existing, but did we really mean for the title to not be localized? 😅 Might be worth following up with that.

Reply:
That's true. There will probably be a follow-up to also localize the URLs.

Reply:
The localized URLs need to be added for the header links too.